src/ixdat/exporters/sec_exporter.py

```diff
@@ -40,9 +39,7 @@ def default_export_columns(self):
         )
         return v_list
 
-    aliases = ECExporter.aliases
-
-    def prepare_header_and_data(self, measurement, v_list, tspan):
+    def prepare_header_and_data(self, measurement, v_list, tspan, time_step=None):
         """Do the standard ixdat csv export header preparation, plus SEC stuff.
```

Comment:
Should this docstring refer to the ECExporter methods for explanation or args?

Reply:
yes.